File: PYTHON/anfal's_python/Assignments/Day1/armstrong.py
print "enter the starting num"
start = input()
print "enter the ending num"
end = input()
for num in range(start, end):
    res = 0
    temp = num
    while temp:
        x = temp % 10
        res += x*x*x
        temp = temp / 10
    
    if num == res:
        print "armstrong number:",num
    

File: C/Manipulations/misc/own/armstrong.c
#include <stdio.h>
int main()
{
        int num = 157,temp,rem;
        int sum =0;
        while(num)
        {
                rem = num%10;
                sum = sum +rem*rem*rem;
                num = num/10;
        }
        if(temp == sum)
                printf("%d is armstrong number...\n",sum);
        return 0;
}
